protocol-definitions/Cache.yaml

```diff
@@ -5,7 +5,8 @@ methods:
     name: addEntryListener
     since: 2.0
     doc: |
-      TODO DOC
+      Adds an entry listener for this cache. The listener will be notified
+      for all cache created/removed/updated/expired events.
```

Comment:
better not to list the events here but refer to the below events list, they are more complete there.